src/AlgAss/AlgGrp.jl

```diff
@@ -167,24 +167,24 @@ function show(io::IO, ::MIME"text/plain", A::GroupAlgebra)
 end
 
 function show(io::IO, A::GroupAlgebra)
+  io = pretty(io)
   if is_terse(io)
     print(io, "Group algebra of ")
     if is_finite(group(A))
-      print(io, "dimension ", order(group(A)))
+      print(io, "dimension ", order(group(A)), " ")
     else
       print(io, "infinite dimension ")
     end
-    print(io, " over ", base_ring(A))
   else
     print(io, "Group algebra of group ")
     if is_finite(group(A))
       print(io, "of order ", order(group(A)), " ")
     else
       print(io, "of infinite order ")
     end
-    print(io, "over ")
-    print(terse(io), base_ring(A))
+    io = terse(io)
```

Comment:
I don't remember what `terse(io)` does, but this looks odd (because we checked `is_terse(io)`?)

Reply:
This is in the `!is_terse` path; it ensures that after the `if/else/end`, the `io` stream definitely is in terse mode. An alternative would be to change the line 187 to
```
  print(terse(io), "over ", Lowercase(), base_ring(A))
```

Reply:
I parse it wrong, my bad